modules/core/05-port/types/stack.go

```diff
@@ -0,0 +1,51 @@
+package types
+
+type IBCStackBuilder struct {
```

Comment:
There is an opportunity to give this extra value by adding a query to ibc core (or somewhere where it makes sense) for users to see the stack. Skip go currently keeps a manual repository of which chains support PFM, for instance. If they could just query, it would help them automate this in the future.
Additionally, chain devs could debug their stack more easily.

It could either just take list here and add another function to the Middleware interface to return a name (types.ModuleName type of response), or maybe even have it come out of ICS4Wrapper so it actually flows through the application stack.
wdyt?

Reply:
This should be a seperate issue and PR if desired